File: CC2 - Midterms (1Y 1S)/Facebook.java
/**
 * @author pao
 * BSCS 1B
 */
import java.util.Scanner;
public class Facebook {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("------------------------\n  Welcome to Facebook  \n------------------------");
        System.out.println("Enter A or a: For log-in\nEnter B or b: For Signup");
        System.out.print("Enter here: ");
        String choice = sc.nextLine();
        final String usernameAdmin = "admin";
        final String passwordAdmin = "admin";
        if (choice.equalsIgnoreCase("a")) {
            System.out.println("--------------------\n  Facebook Log in  \n--------------------");
            System.out.print("Enter username: ");
            String user = sc.nextLine();
            System.out.print("Enter password: ");
            String pass = sc.nextLine();
            if (user.equals(usernameAdmin)){
                if (pass.equals(passwordAdmin)) {
                    System.out.println("Login success");
                    System.out.println("        -----------\n          Profile  \n        -----------");
                    System.out.println("Username      : admin");
                    System.out.println("Email         : admin@gmail.com");
                    System.out.println("Contact Number: 09211234292");
                } else {System.out.println("Invalid password\nPlease try again");}
            } else {System.out.println("Invalid username\nPlease try again");}
        } else if (choice.equalsIgnoreCase("b")) {
            System.out.println("---------------------\n  Facebook Sign Up  \n--------------------");
            System.out.print("Enter username: ");
            String newUser = sc.nextLine();
            System.out.print("Enter password: ");
            String newPass = sc.nextLine();
            System.out.print("Email Address: ");
            String newEmail = sc.nextLine();
            System.out.print("Contact Number: ");
            String newContact = sc.nextLine();
            System.out.println("Sign Up Success\nProceed to Log In");
            System.out.println("--------------------\n  Facebook Log in  \n--------------------");
            System.out.print("Enter username: ");
            String inputUser = sc.nextLine();
            System.out.print("Enter password: ");
            String inputPass = sc.nextLine();
            if (inputUser.equals(newUser)) {
                if (inputPass.equals(newPass)) {
                    System.out.println("Log In Success");
                    System.out.println("        -----------\n          Profile  \n        -----------");
                    System.out.println("Username      : " + newUser);
                    System.out.println("Email         : " + newEmail);
                    System.out.println("Contact Number: " + newContact);
                } else {System.out.println("Invalid Password");}
            } else {System.out.println("Invalid Username");}
        } else {System.out.println("Invalid choice\nPlease try again");}
    }    
}

File: Practice Codes/CC2 Java to Python/Midterms/Exam/Facebook.py
# admin data
USERNAME = "admin"
PASSWORD = "admin"

print("-----------------------")
print("| Welcome to Facebook |")
print("-----------------------")
print()
print("[A/a] Login")
print("[B/b] Signup")
choice = input("Enter here: ").lower()

match choice:
    case "a":
        print("-------------------")
        print("| Facebook Log in |")
        print("-------------------")

        username = input("Enter username: ")
        password = input("Enter password: ")

        if username == USERNAME:
            if password == PASSWORD:
                print("Login success")
                print("---------------")
                print("    PROFILE    ")
                print("---------------")

                print("Username      : admin")
                print("Email         : admin@gmail.com")
                print("Contact Number: 09123456789")
            else:
                print("Invalid password. Try again")
        else:
            print("Invalid username. Try again")

    case "b":
        print("-------------------")
        print("| Facebook Signup |")
        print("-------------------")

        new_username = input("Enter username: ")
        new_password = input("Enter password: ")
        new_email = input("Email Address: ")
        new_contact = input("Contact Number: ")
        print("Sign up Success")
        print("Proceed to Login")
        print()

        print("-------------------")
        print("| Facebook Log in |")
        print("-------------------")

        username = input("Enter username: ")
        password = input("Enter password: ")

        if username == new_username:
            if password == new_password:
                print("Login success")
                print("---------------")
                print("    PROFILE    ")
                print("---------------")

                print("Username:", new_username)
                print("Email Address", new_email)
                print("Contact Number: ", new_contact)
            else:
                print("Invalid password. Try again")
        else:
            print("Invalid username. Try again")
    case _:
        print("Invalid choice. Try again")
